command/server_test.go

```diff
@@ -32,23 +33,36 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type Modifier func(string) string
+
+func regexModifier(re, repl string) Modifier {
```

Comment:
Apologies for the pedantry, but I'm wondering if we can call this something closer to what it does, like maybe `regexReplacer`? 🤔 Names are hard, and I try not to nitpick, but after I had a chin scratching moment reading through the tests, figured it might be worth calling out.

Reply:
I can see that. `regexReplacer` makes sense to me.